lib/models/application.js

```diff
@@ -1562,6 +1565,141 @@ const getApplicationModel = function (deps, opts) {
 			 */
 			remove: envVarModel.remove,
 		},
+
+		/**
+		 * @namespace balena.models.application.invite
+		 * @memberof balena.models.application
+		 */
+		invite: {
+			/**
+			 * @summary Get a single invite
+			 * @name get
+			 * @public
+			 * @function
+			 * @memberof balena.models.application.invite
+			 *
+			 * @param {Number} id - invite id
+			 * @param {Object} [options={}] - extra pine options to use
+			 * @fulfil {Object} - invite
+			 * @returns {Promise}
+			 *
+			 * @example
+			 * balena.models.application.invite.get(51).then(function(invite) {
```

Comment:
I remember that you also had callback examples didn't you?
Just to be clear I never meant to remove the callback examples, just the implementation part.

Reply:
@sradevski mentioned that they are no longer relevant and are noise so removed them.